### PR TITLE
kvm: upgrade libvirt-java to v0.5.3

### PR DESCRIPTION
**Pull request by rohityadavcloud**
Reference: https://gitlab.com/libvirt/libvirt-java/-/issues/6

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)

### CODE REVIEW

**Comment by rohityadavcloud** [comment]
Minor nit - jar is at https://libvirt.org/maven2/org/libvirt/libvirt/ but not sync'd yet on https://mvnrepository.com/artifact/org.libvirt/libvirt

**Review by sureshanaparti — APPROVED** [review]
LGTM

**Comment by rohityadavcloud** [comment]
@blueorangutan package

**Review by GabrielBrascher — APPROVED** [review]
@rohityadavcloud I assume that we should merge only when it is synced in https://mvnrepository.com/artifact/org.libvirt/libvirt.

LGTM, looking forward for libvirt-java 0.5.3 :smiley:

**Comment by rohityadavcloud** [comment]
@GabrielBrascher not necessary, if the upstream project publishes their own repo, we can add it in pom.xml.

File: pom.xml [hunk 1]
```diff
@@ -764,6 +764,10 @@
             <id>repo.jenkins-ci.org.public</id>
             <url>https://repo.jenkins-ci.org/public/</url>
         </repository>
+        <repository>
+            <id>libvirt.org.maven2</id>
+            <url>https://libvirt.org/maven2/</url>
```
Comment (rohityadavcloud) [inline pom.xml:769]: cc @GabrielBrascher I've added libvirt's maven repo here from which the version is available.

**Comment by rohityadavcloud** [comment]
@blueorangutan test

**Review by harikrishna-patnala — APPROVED** [review]
LGMT

**Pull request merged**